install_radar_deps.py

```diff
@@ -303,102 +313,113 @@ def fix_boost_links():
     fix_boost_links()
 
     uhd_cmd = "cd ${IDIR};" \
-    "git clone --recursive git://github.com/EttusResearch/uhd.git;" \
-    "cd uhd || exit;" \
-    "git checkout UHD-4.0;" \
-    "git submodule init;" \
-    "git submodule update;" \
-    "cd host || exit;" \
-    "mkdir build;" \
-    "cd build || exit;" \
-    "cmake -DENABLE_PYTHON3=on -DPYTHON_EXECUTABLE=$(which python3) -DRUNTIME_PYTHON_EXECUTABLE=$(which python3) -DENABLE_PYTHON_API=ON -DENABLE_DPDK=OFF ../;" \
-    "make -j${CORES};" \
-    "make -j${CORES} test;" \
-    "make install;" \
-    "ldconfig;"
+              "git clone --recursive https://github.com/EttusResearch/uhd.git;" \
+              "cd uhd || exit;" \
+              "git checkout UHD-4.0;" \
+              "git submodule init;" \
+              "git submodule update;" \
+              "cd host || exit;" \
+              "mkdir build;" \
+              "cd build || exit;" \
+              "cmake -DENABLE_PYTHON3=on -DPYTHON_EXECUTABLE=$(which python3) " \
+              "-DRUNTIME_PYTHON_EXECUTABLE=$(which python3) -DENABLE_PYTHON_API=ON -DENABLE_DPDK=OFF ../;" \
+              "make -j${CORES};" \
+              "make -j${CORES} test;" \
+              "make install;" \
+              "ldconfig;"
 
     execute_cmd(uhd_cmd)
 
+
 def install_cuda():
```

Comment:
We need to reconsider how CUDA is installed. This is a constant problem on site visits when installing new computers. We should add the creation of the video user group here.

Reply:
You have had better success installing CUDA than I have, care to take a crack at it?

Reply:
ok